dev/devicelab/lib/framework/framework.dart

```diff
@@ -82,7 +83,33 @@ class _TaskRunner {
     try {
       _taskStarted = true;
       print('Running task.');
-      final TaskResult result = await _performTask().timeout(taskTimeout);
+      final String exe = Platform.isWindows ? '.exe' : '';
+      section('Checking running Dart$exe processes');
+      final Set<RunningProcessInfo> beforeRunningDartInstances = await getRunningProcesses(
+        processName: 'dart$exe',
+      ).toSet();
+      beforeRunningDartInstances.forEach(print);
+
+      TaskResult result = await _performTask().timeout(taskTimeout);
+
+      section('Checking running Dart$exe processes after task...');
+      final List<RunningProcessInfo> afterRunningDartInstances = await getRunningProcesses(
+        processName: 'dart$exe',
+      ).toList();
+      afterRunningDartInstances.forEach(print);
+      for(final RunningProcessInfo info in afterRunningDartInstances) {
+        if (!beforeRunningDartInstances.contains(info)) {
```

Comment:
Above prints seem redundant with this

Reply:
This one is printing out exactly why we thought we had a matching process.  Maybe I'm being paranoid, but this should only add ~5-10 lines extra to the logs.